lib/internal/bootstrap_node.js

```diff
@@ -190,6 +190,12 @@
   }
 
   function setupGlobalVariables() {
+    Object.defineProperty(global, 'global', {
```

Comment:
(I'd definitely prefer to define it as non-enumerable in C++ instead, to be sure)